apps/condo/domains/acquiring/access/GeneratePaymentLinkService.js

```diff
@@ -2,10 +2,12 @@
  * Generated by `createservice acquiring.GeneratePaymentLinkService`
  */
 const { throwAuthenticationError } = require('@condo/keystone/apolloErrorFormatter')
+const { STAFF, SERVICE } = require('@condo/domains/user/constants/common')
 
 async function canGeneratePaymentLink ({ authentication: { item: user } }) {
     if (!user) return throwAuthenticationError()
-    return true
+
+    return user.isAdmin || user.type === STAFF || user.type === SERVICE
```

Comment:
user.type !== RESIDENT ?

Reply:
A link supposed to be generate by organisation stuff / admin. A resident doesn't supposed to generate a link for himself, at least he should not be involved in receipt generation process and subprocesses (like receipt link generation).